addons/html_editor/static/src/core/format_plugin.js

```diff
@@ -161,9 +180,90 @@ export class FormatPlugin extends Plugin {
             this._formatSelection(format, { applyStyle: false });
         }
         this.dispatchTo("remove_format_handlers");
+    }
+
+    removeFormat() {
+        this._removeFormat();
         this.dependencies.history.addStep();
     }
 
+    paintFormat() {
+        if (this._copiedFormats.length) {
+            this._paintFormat();
+        } else {
+            this._copyFormat();
```

Comment:
I would call these `pasteFormat` and `copyFormat`. What do you think? Also, there is no need to use underscore to prefix methods in plugins since all methods are protected by default unless they are specified in `shared`.